README.rst

```diff
@@ -69,7 +69,7 @@ If you are using multiple Travis factors,
 then you can use those factors to decide what will run.
 For example, see the following ``.travis.yml`` and ``tox.ini``:
 
-.. code-block::
+.. code-block:: yaml
```

Comment:
Awesome. I'll check to make sure when I release, but I'll be happy to have the highlighting added!